mods/common/chrome/color-picker.yaml

```diff
@@ -42,3 +42,23 @@ Background@COLOR_CHOOSER:
 			Y: 21
 			Width: 80
 			Height: 73
+		Label@PRESETCOLORSLABEL:
```

Comment:
Having everything on the one panel looks awkward IMO.
Can you put these into a separate tab instead?  Here is an old mockup from when the color picker was originally being built showing how it could look:

![htoxo14](https://user-images.githubusercontent.com/167819/36641583-b65b979c-1a29-11e8-8a1f-ca00f2b46fe3.png)

Reply:
Thanks for the feedback. I'll give this another try.